src/Microsoft.DotNet.XHarness.CLI/Commands/Apple/AppleUninstallCommand.cs

```diff
@@ -28,12 +28,12 @@ public AppleUninstallCommand(IServiceCollection services) : base("uninstall", fa
                 .BuildServiceProvider()
                 .GetRequiredService<IUninstallOrchestrator>()
                 .OrchestrateAppUninstall(
-                    AppleAppArguments.BundleIdentifier,
-                    AppleAppArguments.Target,
-                    AppleAppArguments.DeviceName,
-                    AppleAppArguments.Timeout,
-                    AppleAppArguments.ResetSimulator,
-                    AppleAppArguments.EnableLldb,
+                    Arguments.BundleIdentifier,
+                    Arguments.Target,
+                    Arguments.DeviceName,
+                    Arguments.Timeout,
+                    false,
```

Comment:
just curious, why the change to constant ?

Reply:
Before, we were overriding the argument value to `false` because we couldn't hide the member completely thanks to inheritance